src/Nest/Modules/Scripting/IStoredScript.cs

```diff
@@ -1,74 +1,37 @@
-﻿using System.Collections.Generic;
-using Elasticsearch.Net;
-using Newtonsoft.Json;
+﻿using Newtonsoft.Json;
 
 namespace Nest
 {
 	[JsonObject(MemberSerialization = MemberSerialization.OptIn)]
 	[JsonConverter(typeof(ReadAsTypeJsonConverter<StoredScript>))]
 	public interface IStoredScript
 	{
-		[JsonProperty("lang")]
```

Comment:
You can still specify lang as part of the body, its just no longer part of the path in the url:

https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/script/StoredScriptSource.java#L157